numba/np/arraymath.py

```diff
@@ -4722,3 +4722,36 @@ def impl(a, b):
         return _cross2d_operation(a_, b_)
 
     return impl
+
+
+@overload(np.trim_zeros)
+def np_trim_zeros(a, trim='fb'):
+    if not isinstance(a, types.Array):
+        raise NumbaTypeError('The first argument must be an array-like')
```

Comment:
I think it should just be an array now, not array-like?

```suggestion
        raise NumbaTypeError('The first argument must be an array')
```